scripts/setlocalversion

```diff
@@ -3,12 +3,16 @@
 # Print additional version information for non-release trees.
 
 usage() {
-	echo "Usage: $0 [srctree]" >&2
+	echo "Usage: $0 [srctree] [version]" >&2
 	exit 1
 }
 
 cd "${1:-.}" || usage
 
+if [ -n "${2}" ]; then
+        printf '%s' "${2}"
```

Comment:
Uh... isn't this doing nothing but echoing back the second argument if given.  Why is that useful?